pom.xml

```diff
@@ -69,6 +70,8 @@
     <opentelemetry-semconv.version>1.25.0-alpha</opentelemetry-semconv.version>
     <!-- from grpc-protobuf -->
     <protobuf.version>3.25.1</protobuf.version>
+    <!-- Netty -->
+    <netty.version>4.1.108.Final</netty.version>
```

Comment:
Is this property needed actually?

Reply:
Removed the leftover in https://github.com/openzipkin-contrib/zipkin-otel/pull/9/commits/1f6b0299837629e5a4ccb756f1d74831d59bc93d